AlertaDengue/dados/templates/alert_base.html

```diff
@@ -244,7 +244,29 @@ <h4>{% blocktrans %} Últimas {{ look_back }} semanas: {% endblocktrans %}</h4>
 
         <div class="col-md-3 bluebox p-3">
             <h3><span class="glyphicon glyphicon-map-marker" aria-hidden="true"></span> {% blocktrans %}Suspeita de um criadouro?{% endblocktrans %}</h3>
-            <p>{% blocktrans %}Em caso de suspeita de criadouro de mosquitos, ligue para a <a href="http://www.1746.rio.gov.br/" style="color: #FFFFFF">Central de atendimento ao cidadão 1746</a>.{% endblocktrans %}</p>
+            <p>
+            
+            {% blocktrans %}Em caso de suspeita de criadouro de mosquitos, {% endblocktrans %}
+            <!-- RJ -->    
+            {% if state == 'Rio de Janeiro' %}
+                {% blocktrans %}contacte a central de apoio ao cidadão: 1746{% endblocktrans %}
```

Comment:
it is missing the link to `http://www.1746.rio.gov.br/` check here:

- https://info.dengue.mat.br/alerta/rio/dengue

at https://github.com/AlertaDengue/AlertaDengue/issues/179 ... in the specific description there is no link but @claudia-codeco says on Objective that it is correctly just for Rio city ... so I think you should add the link.

Reply:
Yes this link is only valid for Rio city. For other places we can either try to find out if they have a service for reporting breeding sites or not including the link.

Reply:
Fiquei em dúvida se é ou não para adicionar o link.

Reply:
ok

Reply:
O mais urgente aqui é trocar o texto de todos os municipios exceto o Rio de Janeiro.